cmd/sonobuoy/app/images.go

```diff
@@ -101,3 +153,205 @@ func getImages(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 }
+
+func pullImages(cmd *cobra.Command, args []string) {
+	switch imagesflags.plugin {
+	case "e2e":
+
+		cfg, err := imagesflags.kubeconfig.Get()
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't get REST client"))
+			os.Exit(1)
+		}
+
+		sbc, err := getSonobuoyClient(cfg)
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "could not create sonobuoy client"))
+			os.Exit(1)
+		}
+
+		version, err := sbc.Version()
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't get Sonobuoy client"))
+			os.Exit(1)
+		}
+
+		upstreamImages, err := image.GetImages("", version)
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't init upstream registry list"))
+			os.Exit(1)
+		}
+
+		for _, v := range upstreamImages {
+			err := docker.PullIfNotPresent(v.GetE2EImage(), numDockerRetries)
+			if err != nil {
+				errlog.LogError(errors.Wrapf(err, "couldn't pull image: %v", v.GetE2EImage()))
+			}
+		}
+	default:
+		errlog.LogError(errors.Errorf("Unsupported plugin: %v", imagesflags.plugin))
+		os.Exit(1)
+	}
+}
+
+func downloadImages(cmd *cobra.Command, args []string) {
+	switch imagesflags.plugin {
+	case "e2e":
+
+		cfg, err := imagesflags.kubeconfig.Get()
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't get REST client"))
+			os.Exit(1)
+		}
+
+		sbc, err := getSonobuoyClient(cfg)
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "could not create sonobuoy client"))
+			os.Exit(1)
+		}
+
+		version, err := sbc.Version()
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't get Sonobuoy client"))
+			os.Exit(1)
+		}
+
+		upstreamImages, err := image.GetImages("", version)
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't init upstream registry list"))
+			os.Exit(1)
+		}
+
+		images := []string{}
+		for _, v := range upstreamImages {
+			images = append(images, v.GetE2EImage())
+		}
+
+		fileName := image.GetTarFileName(version)
+		err = docker.Save(images, fileName)
+		if err != nil {
+			errlog.LogError(errors.Wrap(err, "couldn't save images to tar"))
+			os.Exit(1)
+		}
+
+		fmt.Println("Archive is: ", fileName)
```

Comment:
Although the prefix makes it easy to read I think the benefit of outputting this at all is being able to easily script around it. 

Without the prefix you just need to know the output is the file:
```
output=$(sonobuoy images download)
```

With the prefix you have to know the format of the output and use another tool like awk or cut:
```
output=$(sonobuoy images download | awk '{print $NF}'
```

Removing the prefix would also make it consistent with `sonobuoy retrieve`.